src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -413,6 +414,26 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 set: (el, value) => el.setAttribute('manifest', value)
             },
 
+            onload: {
+                condition: el => domUtils.isDomElement(el) && domUtils.isImgElement(el),
```

Comment:
We need a comment which briefly describes why we need it and points out to the github discussion